index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Project -1</title>
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;500&family=Roboto:wght@100&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <header class = "navheader">
+        <div class="logo"><a href="#" target="_blank" class="logo-link">My Logo</a></div>
+        <nav class="menubar">
+            <div class ="menu1"></div>
+            <div class ="menu2"></div>
+            <div class ="menu3"></div>
+        </nav>
+    </header>
+
+    <div class="shapespace">
+        <div class="rectangle6"></div>
+        <div class="eclipse11"></div>
+        <div class="eclipse8"></div>
+        <div class="rectangle7"></div>
+        <div class="ellipse5"></div>
+        <div class="ellipse12"></div>
+        <div class="ellipse15"></div>
+        <div class="U"></div>
+        <div class="ellipse16"></div>
+        <div class="eclipse3"></div>
+        <div class="rectangle4"></div>
+        <div class="finalrect"></div>
+        <div class="circle"></div>
+
+    </div>
+
+    <div class = "main">
+        <h1 class="mainfont"> I'm Mukund <br> Glad to see <br> you!</h1>
+        <p class = "mainpara">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href = "#" target="_self" class="connect">LET'S CONNECT</a>
+    </div>
+
+    <section class="logo">
+    <ul class="image">
+     <li class="images"><a href="#" id="twitter"></a></li>
+     <li class="images"><a href="#" id="In"></a></li>
+     <li class="images"><a href="#" id="M"></a></li>  
+     <li class="images"><a href="#" id="Github"></a></a><li>   
+    <li class="images"><a href="#" id="Hand"></a> </li>
+    </ul>
+    </section>
```

Comment:
- [OPTIONAL ] I noticed that your social media profiles are not visible in your project; please include them so that recruiters may easily interact with you. :smile: :upside_down_face: